src/js/components/Chart/doc.js

```diff
@@ -201,14 +209,14 @@ export const themeDoc = {
         responsiveBreakpoint: 'small',
     }`,
   },
-  'global.graph.colors': {
-    description: 'The color to use when not specified via color.',
-    type: '[string] or { dark: [string], light: [string] }',
-  },
   'global.opacity': {
     description: 'The opacity of the Chart stroke.',
     type: 'string',
-    defaultValue: undefined,
+    defaultValue: `{
+      strong: 0.8,
+      medium: 0.4,
+      weak: 0.1,
```

Comment:
Not sure I understand which one is the default though?

Reply:
Well, the `global.opacity` property have a value that is an object, as shown. I was following the examples I was seeing for `global.edgeSize`.

Reply:
correct, but in some cases when we actually have a default value we will add something like (assuming the default is `medium`):
```
  'global.opacity.medium': {
    description: 'The opacity value used for the Chart color.',
    type: 'number',
    defaultValue: '0.4',
  },
```

Reply:
I think the more interesting part is what the default value for the `opacity` property of Chart is, which is undefined/false. I thought showing all opacity values from the theme here aligned better with how the `opacity` property is documented. And, the default value of the `opacity` property should be documented with the component property, not with the theme.

Reply:
I understand your point.